resources/check.rb

```diff
@@ -6,7 +6,8 @@
 attribute :timeout, :kind_of => Integer
 attribute :subscribers, :kind_of => Array
 attribute :standalone, :kind_of => [TrueClass, FalseClass]
-attribute :aggregate, :kind_of => [TrueClass, FalseClass]
+attribute :aggregate, :kind_of => String
```

Comment:
@Evesy per https://github.com/sensu/sensu-settings/blob/v6.0.0/lib/sensu/settings/validators/check.rb#L108-L116 I think this should be `:kind_of => [String, TrueClass, FalseClass]`